File: 07_covid home/200317_Collection/example#1/java/before.java
import java.util.Random;

public class Main
{
    
	public static String getFruit() {
        
		int random = (new Random()).nextInt() & Integer.MAX_VALUE;
        
		int i = random % 4;
        
		switch(i) {
            
			case 0:
				return "Mango";
			case 1:
				return "Apple";
			case 2:
				return "Orange";
			case 3:
				return "Banana";
			default:
				return null;
		}
	}
    
	public static void main(String[] args) {
	    
		System.out.println(getFruit());
	}
}

File: 07_covid home/200317_Collection/example#1/python/before.py
import random


def getFruit():
    
    fruits = ["Mango", "Orange", "Apple", "Banana"]
    
    i = random.randrange(0, 4)
    
    if i == 0:
        return "Mango"
    elif i == 1:
        return "Orange"
    elif i == 2:
        return "Apple"
    elif i == 3:
        return "Banana"

print(getFruit())

